packages/webapp/src/containers/Finances/NewExpense/AddExpense/index.jsx

```diff
@@ -200,10 +205,11 @@ class AddExpense extends Component {
                     ))}
                   </div>
                   <div className={styles.addContainer}>
-                    <div className={styles.greenPlus}>+</div>
-                    <button onClick={() => this.addSubExpense(k)}>
-                      {this.props.t('EXPENSE.ADD_MORE_ITEMS')}
-                    </button>
+                    <Tooltip title={this.props.t('EXPENSE.ADD_MORE_ITEMS')}>
+                      <div className={styles.greenPlus} onClick={() => this.addSubExpense(k)}>
```

Comment:
The ticket says to add new button style. I don't think that has happened?

Reply:
I need to consult with Sasha.